Added description (#670)
demisto/content-docs: docs/packs/packs-format.md

The image size should be up to 4kb and in the dimensions of 120x50.
 
In case file does not exist, the pack author will be displayed in text.

> 

Reviewer: ```suggestion
```

### CONTRIBUTORS.md

> If you are editing existing pack, you can add **CONTRIBUTORS.md** file to the pack, and add you name. This file's content will appear at the pack description at the Marketplace.

Reviewer: ```suggestion
If you are contributing to an existing pack, you can add a **CONTRIBUTORS.md** file to the pack and list yourself as a contributor for the pack. The file's content will appear in the pack README in XSOAR Marketplace.
```

#### Example CONTRIBUTORS.md
```
### Pack Contributors:

---

>  - Contributor1
>  - Contributor2

Reviewer: ```suggestion
 - Jane Doe
 - John Smith
```
